sympy/parsing/latex/__init__.py

```diff
@@ -31,11 +42,21 @@ def parse_latex(s, strict=False):
     (sqrt(a) + 1)/b
     >>> expr.evalf(4, subs=dict(a=5, b=2))
     1.618
+    >>> func = parse_latex(r"\int_1^\alpha \dfrac{\mathrm{d}t}{t}", backend="Lark")
+    >>> func.evalf(subs={"alpha": 2})
+    0.693147180559945
     """
 
-    _latex = import_module(
-        'sympy.parsing.latex._parse_latex_antlr',
-        import_kwargs={'fromlist': ['X']})
-
-    if _latex is not None:
-        return _latex.parse_latex(s, strict)
+    backend = backend.lower()
```

Comment:
Do we need to lowercase the enumerative arguments?

Reply:
I'm not sure I understood your question. What exactly are you objecting to here?

Reply:
Why do we need to check `backend.lower()` than `backend == 'antlr' or backend == 'lark'`?

To get my context why I support checking exact argument, because in the AI era (where language models and autocompletion tools dominates the programmers) 
I don't think that it would be very valuable to write logic by hand, that recognize `antlr, AnTlR, ANTLR, Antlr, ...`. 
I may do it on my app, but I won't do that for library code.

Of course, I don't have strong preference what others do for it, but I would ask why you would opinionate to do it.

Reply:
Fair enough. I'll make the change when I get back to my laptop. I'll be away from my laptop for a few days so I won't be able to resolve this right away.

Reply:
I've made the change.